daemon/volumes.go

```diff
@@ -119,7 +119,15 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 	}
 
-	// 3. Read bind mounts
+	// 3. Read tmpfs
+	for destination := range hostConfig.Tmpfs {
+		mountPoints[destination] = &volume.MountPoint{
+			Source:      "tmpfs",
```

Comment:
had a fast look around - I'm not sure how to achieve the same w/o appending something to the mounts here..